### PR TITLE
[djangocon-2015] Add missing speakers

### PR DESCRIPTION
**Pull request by redapple**
Continuation of #47 + https://github.com/pyvideo/pyvideo-data/pull/47#issuecomment-218805560

### CODE REVIEW

**Comment by willkg** [comment]
I think we should nix the speaker names from the title. If we ever have to fix speaker names later on, it'll make it easier if we only have to look at the speakers field.

Can you do that? If not, I can find some time to do it or throw together an issue for it.

**Comment by redapple** [comment]
@willkg , done.

File: data/djangocon-2015/videos/beyond-the-basics-with-elasticsearch-by-honza-kral.json [hunk 1]
```diff
@@ -12,12 +12,14 @@
...
-      "length": 0
+      "length": 0,
+      "type": "youtube"
     }
   ],
   "source_url": "https://www.youtube.com/watch?v=RAJiFK1bHO0",
   "tags": [],
-  "speakers": [],
+  "speakers": [
+    "Honza Kral"
```
Comment (willkg) [inline data/djangocon-2015/videos/beyond-the-basics-with-elasticsearch-by-honza-kral.json:22]: I noticed that Honza's name is spelled two ways across our data. This is the all-ascii way. I'll write up an issue to fix his name in the places where it's all-ascii.

File: data/djangocon-2015/videos/building-theatlantic-com-homepages-wysiwyg-admin.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "category": "",
   "slug": "building-theatlantic-com-homepages-wysiwyg-admin",
-  "title": "Building theatlantic.com homepage\u2019s WYSIWYG admin with... by Frankie Dintino",
+  "title": "Building theatlantic.com homepage’s WYSIWYG admin with Django and Knockout",
```
Comment (willkg) [inline data/djangocon-2015/videos/building-theatlantic-com-homepages-wysiwyg-admin.json:4]: Awesome fixing this!

File: data/djangocon-2015/videos/django-authors-panel.json [hunk 3]
```diff
@@ -12,12 +12,19 @@
...
   "source_url": "https://www.youtube.com/watch?v=Ulv_ygVuhwI",
   "tags": [],
-  "speakers": [],
+  "speakers": [
+    "Mark Lavin",
+    "Andrew Pinkham",
+    "Buddy Lindsey",
+    "Peter Baumgartner",
+    "Rikki Endsley",
+    "Tracy Osborn"
```
Comment (willkg) [inline data/djangocon-2015/videos/django-authors-panel.json:27]: You rock! THANK YOU!

**Comment by willkg** [comment]
Thank you so much for your diligence here. This is great work!

File: data/djangocon-2015/videos/building-theatlantic-com-homepages-wysiwyg-admin.json (hunk 2 above)
Comment (redapple) [inline data/djangocon-2015/videos/building-theatlantic-com-homepages-wysiwyg-admin.json:4]: Actually, it's probably better to use `\u2019` in the JSON (I copy/pasted the title from the web page, without checking non-ASCII characters).
I'll fix this

**Pull request merged**